packages/application/src/application/services/role.service.ts

```diff
@@ -1,23 +1,28 @@
 import { inject, injectable } from 'inversify';
 import config from '@src/config';
-import { GovernanceConfig } from '@src/infrastructure/interfaces';
+import { GovernanceConfig, MetaAllocatorConfig } from '@src/infrastructure/interfaces';
 import { TYPES } from '@src/types';
+import { MetaAllocatorService } from './meta-allocator.service';
 
 const RKH_ADDRESSES = config.RKH_ADDRESSES;
-const MA_ADDRESSES = config.MA_ADDRESSES;
 
 @injectable()
 export class RoleService {
   constructor(
     @inject(TYPES.GovernanceConfig) private readonly governanceConfig: GovernanceConfig,
+    @inject(TYPES.MetaAllocatorConfig) private readonly metaAllocatorConfig: MetaAllocatorConfig,
+    @inject(TYPES.MetaAllocatorService) private readonly metaAllocatorService: MetaAllocatorService,
   ) {}
   getRole(address: string): string {
     let role = 'USER';
     if (this.governanceConfig.addresses.includes(address.toLowerCase())) {
       role = 'GOVERNANCE_TEAM';
     } else if (RKH_ADDRESSES.includes(address.toLowerCase())) {
       role = 'ROOT_KEY_HOLDER';
-    } else if (MA_ADDRESSES.includes(address.toLowerCase())) {
+    } else if (
+      this.metaAllocatorConfig.signers.includes(address.toLowerCase()) ||
```

Comment:
Just to be clear on the terminology: we are now calling it "signers" but it's still actually 0x... addresses, like the old environment variable?

Also is this ready for multiple MetaAllocators? They're just all lumped in together?